### DISCUSSION TITLE
Allow maintiners to select default update method for open pull requests

### DISCUSSION THREAD

**Discussion by sithlord48** [community/community#127943] [Repositories]
### Select Topic Area

Product Feedback

### Body

When a Pull request is out of date github provide an "update" button that by default does a merge commit to update the Pull request. There is also the option to use a "rebase update" if you click the drop down on this button like below

<img width="984" alt="Screenshot 2024-06-09 at 5 49 43 PM" src="https://github.com/input-leap/input-leap/assets/7450820/b0ff9b2b-1fa3-4ec5-bc50-424a923f98db">

It would be very useful for maintainers to be able to select what the default action is. Better yet to allow us to restrict this to just the option the maintainers want contributors to use when updating their stale pull requests

**Comment by shymega** [comment]
To second this - for example of the functionality we'd like on Input Leap, please see: https://github.com/input-leap/input-leap/pull/1830.
